From 9b17b085f3dbe1e48dfea2ed1b045fbbbc10d3d9 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Wed, 14 Apr 2010 17:58:56 +0200
Subject: [PATCH 09/15] manpage addition for kernel drivers

The kernel modules have to be specified without the ".ko" suffix.
---
 dracut.8      |    2 ++
 dracut.conf.5 |    2 ++
 2 files changed, 4 insertions(+), 0 deletions(-)

diff --git a/dracut.8 b/dracut.8
index 834c4f2..31a70fc 100644
--- a/dracut.8
+++ b/dracut.8
@@ -30,9 +30,11 @@ add a space-separated list of dracut modules.
 .BR \-d ", " \-\-drivers " \fILIST\fR"
 specify a space-separated list of kernel modules to exclusively include
 in the initramfs.
+The kernel modules have to be specified without the ".ko" suffix.
 .TP
 .BR \-\-add-drivers " \fILIST\fR"
 specify a space-separated list of kernel modules to add to the initramfs.
+The kernel modules have to be specified without the ".ko" suffix.
 .TP
 .BR \-\-filesystems " \fILIST\fR"
 specify a space-separated list of kernel filesystem modules to exclusively 
diff --git a/dracut.conf.5 b/dracut.conf.5
index 30bf16f..a781dfb 100644
--- a/dracut.conf.5
+++ b/dracut.conf.5
@@ -26,10 +26,12 @@ Add a space-separated list of dracut modules.
 .BR drivers= \%"[LIST]"
 Specify a space-separated list of kernel modules to
 exclusively include in the initramfs.
+The kernel modules have to be specified without the ".ko" suffix.
 .TP
 .BR add_drivers= \%"[LIST]"
 Specify a space-separated list of kernel 
 modules to add to the initramfs.
+The kernel modules have to be specified without the ".ko" suffix.
 .TP
 .BR filesystems= \%"[LIST]"
 Specify a space-separated list of kernel filesystem
-- 
1.7.0.1

